packages/x-charts-material/src/FakeCss/index.ts

```diff
@@ -0,0 +1,9 @@
+import { styled } from '@mui/material';
+
+export const FakeCss = styled('div')(({ theme }) => ({
```

Comment:
Can't we just wrap the PieChart.Root component and provide all the styles here? Why FakeCSS? This is Material UI package, no?